### PR TITLE
Feature/form additional: 4th part and last of the form :)

### PR DESCRIPTION
**Pull request by orjwan98**
References issues #89 #78 

References PR #102 

What I did in this PR: 

- [x] Make the AdditionalInfo component which is the last part of the form. 

- [x] Make handlers to its parent component and pass them down to it as props. 

- [x] Set state for it in the parent component and pass them down to it as props. 

- [x] Make a data.json file that contains categories and tags. (note:categories are mockdata).

### CODE REVIEW

File: client/src/Components/Form/AdditionalInfo/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <Select
+          aria-label="category"
+          value={category}
+          onChange={categoryChange}
+          inputProps={{
+            name: 'category',
+            id: 'category',
+          }}
+        >
+          {arrCategory.map(cat => (
```
Comment (teenie-quaggard) [inline client/src/Components/Form/AdditionalInfo/index.js]: needs a ```key``` prop

**Comment by teenie-quaggard** [comment]
@orjwan98 one very small change requested! but otherwise looks good 🦄

**Comment by orjwan98** [comment]
@teenie-quaggard I just pushed my last changes, it's ready to merge if you're happy with it c:

**Comment by orjwan98** [comment]
@teenie-quaggard okay now I fixed everything for real :D And removed all errors and warnings. c:

**Comment by teenie-quaggard** [comment]
@orjwan98 
yay1

**Pull request merged**